devbox.go

```diff
@@ -379,6 +381,73 @@ func (d *Devbox) Info(pkg string) error {
 	return pkgcfg.PrintReadme(pkg, d.configDir, d.writer, false /*showSourceEnv*/)
 }
 
+// generates devcontainer.json and Dockerfile for vscode run-in-container
+// and Github Codespaces
+func (d *Devbox) GenerateDevcontainer(force bool) error {
+	// construct path to devcontainer directory
+	devContainerPath := filepath.Join(d.configDir, ".devcontainer/")
+	devContainerJSONPath := filepath.Join(devContainerPath, "devcontainer.json")
+	dockerfilePath := filepath.Join(devContainerPath, "Dockerfile")
+
+	filesExist := false
+	// check if devcontainer.json doesn't exits
+	if plansdk.FileExists(devContainerJSONPath) {
+		filesExist = true
+	}
```

Comment:
Can this be simplified as filesExist := plansdk.FileExists(devContainerJSONPath)?